src/tools/rest-frontend/public/assets/translations/en.json

```diff
@@ -0,0 +1,370 @@
+    "APP.ENTRIES.DETAILS.NOTIFICATION.CLIPBOARD.MESSAGE.SUCCESS": "The snippet has been copied to the clipboard successfully.",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.APPLICATION": "Application:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.APPSCOPE": "Application scope:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.AUTHOR": "Author:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.CREATEDAT": "Created at:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.KEY": "Full entry:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.ORGANIZATION": "Organization:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.SCOPE": "Scope:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.SLUG": "Slug:",
+    "APP.ENTRIES.DETAILS.TABLE.HEADER.TAGS": "Tags:",
+    
+    "APP.ENTRIES.EDIT.NOTIFICATION.HEADER": "Edit configuration snippet",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_DOES_NOT_EXIST": "The entry you are attempting to change does not exist. It is possible that it has been deleted in the mean time.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_INVALID_CONFIGURATION_FORMAT": "The submitted configuration format is not supported.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_INVALID_CONFIGURATION_VALUE": "The submitted configuration snippet could not be parsed within the submitted format. If you are sure the format is correct, please try one of the other plugins.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_INVALID_DESCRIPTION": "The submitted description does not match the requirements.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_INVALID_TAG": "One or more of the submitted tags does not match the requirements.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_INVALID_TITLE": "The submitted title does not match the requirements.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_MISSING_CONFIGURATION_FORMAT": "You have to submit the format of your snippet.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_MISSING_CONFIGURATION_VALUE": "You have to submit a configuration snippet.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_MISSING_DESCRIPTION": "You have to submit a description for the snippet.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_MISSING_TITLE": "You have to submit a title for the snippet.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.ENTRY_UPDATED_SUCCESSFULLY": "The configuration snippet has been updated successfully. Please check if the import of the new configuration has not caused any issues. In case something is wrong, simply edit the snippet again!",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.NEED_AUTHENTICATION": "You have to be authenticated to perform this action. If the error remains, please refresh the page and login again if necessary.",
+    "APP.ENTRIES.EDIT.NOTIFICATION.MESSAGE.REQUEST_MALFORMED_DATA": "An error occured. Please contact the administrator, if the error persists.",
```

Comment:
JSON does not support that, unfortunately. But as this is one of the messages that no one should ever see, we will also not need to change them.